desc/compute/_core.py

```diff
@@ -1648,10 +1648,10 @@ def _alpha_tz(params, transforms, profiles, data, **kwargs):
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["theta_PEST_z", "phi_z", "iota"],
+    data=["theta_PEST_z", "zeta_z", "iota"],
```

Comment:
I think this is wrong, we need this to be `"phi_z"` because now `zeta=phi` is not assumed, but `alpha` as defined within the PEST coordinate system uses the toroidal angle `phi`

Reply:
Yeah, all the phis and their derivatives in this PR are actually what we call zetas so you are right.

Reply:
Yes we had already changed these from zeta to phi in other PRs.